lib/stackify/utils/methods.rb

```diff
@@ -10,7 +10,7 @@ def self.rounded_current_time
   end
 
   def self.is_mode_on? mode
-    Stackify.configuration.mode == Stackify::MODES[:both] || Stackify.configuration.mode == mode
+    Stackify.configuration.mode = mode || Stackify::MODES[:both]
```

Comment:
what does `::` do?

is it accessing static attribute from `Stackify` class and with `Hash` value?
sorry for the question. no idea how ruby still works. still reading and learning it :)

Reply:
`::`  this is a namespace operator the same symbol being use in PHP. Yes, you can access the Stackify class with the MODES property.